ietf-dtnma-agent.yang

```diff
@@ -642,6 +642,74 @@ module ietf-dtnma-agent {
     }
   }
 
+  // CTRLs related to rules
+  amm:ctrl add-sbr {
```

Comment:
There is no way to define to which ODM the SBR (or TBR) will be added. It may be helpful to provide an additional parameter to allow the ODM to be defined, for example as is done for `VAR` and `CONST`:

```
    amm:parameter obj {
      description
        "A reference to a VAR within an ODM only.";
      amm:type "/ARITYPE/VAR";
    }
```